src/of.js

```diff
@@ -20,4 +19,4 @@ var _of = require('./internal/_of');
  *      R.of(null); //=> [null]
  *      R.of([42]); //=> [[42]]
  */
-module.exports = _curry1(_of);
+module.exports = _of;
```

Comment:
Sorry, the suggestions was just to move the contents of `internal/_of` into here and then remove `internal/_of.js`.